pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyLanguageDao.java

```diff
@@ -41,6 +42,10 @@ default long insert(String studyGuid, String languageCode, String name) {
         return getStudyLanguageSql().insert(studyDto.getId(), languageCodeId, name);
     }
 
+    default int deleteStudyLanguageById(long studyLanguageId) {
```

Comment:
Typically, caller should almost always check the returned value to ensure that indeed only one row is deleted. For convenience I recommend doing that check in this method itself so we lessen the burden on callers. There's the `DBUtils.checkDelete()` method you can use.

We don't have to do it for the method in `StudyLanguageSql` since it's more of a "lower-level" SQL method. But for DAOs, let's add a check.